internal/generator/vector/output/cloudwatch/ca_file.go

```diff
@@ -0,0 +1,17 @@
+package cloudwatch
```

Comment:
Why do we have to duplicate the tls logic instead of reusing something that already exists?

Reply:
I agree that we shouldn't but I kept it in line with the current implementations to all of the outputs. I can refactor the TLS logic for all of the outputs but the scope of this ticket is not that.

Reply:
@jcantrill I've refactored TLS to remove redundancy in the code